base/repl/REPL.jl

```diff
@@ -122,7 +122,7 @@ end
 function display(d::REPLDisplay, mime::MIME"text/plain", x)
     io = outstream(d.repl)
     Base.have_color && write(io, answer_color(d.repl))
-    show(IOContext(io, :limit => true), mime, x)
+    show(IOContext(io, :limit => true, :module => Main), mime, x)
```

Comment:
We may want to allow giving the module via a keyword, but that can be added later.